llvm/lib/SYCLLowerIR/SpecConstants.cpp

```diff
@@ -29,6 +29,19 @@
 
 using namespace llvm;
 
+static cl::opt<SpecConstantsPass::HandlingMode> SpecConstantMode(
```

Comment:
is there a default value that can be set here?

Reply:
added default value: `cl::init(SpecConstantsPass::HandlingMode::emulation)`